package-configuration/linux-cli/get-cloudify.py

```diff
@@ -210,9 +223,17 @@ def install_module(module, version=False, pre=False, virtualenv_path=False,
             IS_VIRTUALENV))
     result = run(' '.join(pip_cmd))
     if not result.returncode == 0:
+        lgr.error(result.aggr_stdout)
         sys.exit('Could not install module: {0}.'.format(module))
 
 
+def untar(archive, destination):
+    """This will extract a tar.gz archive and extract it while stripping
+    the parents directory within the archive.
+    """
+    run('tar -xzvf {0} -C {1} --strip=1'.format(archive, destination))
```

Comment:
why not using `tarfile` instead? https://docs.python.org/2/library/tarfile.html

    import tarfile
    tarfile = tar.open(name='tarfile.tar.gz')
    tar.extractall(path='/dest')